src/mgmt/cli_mgmt.c

```diff
@@ -335,10 +342,13 @@ int dc_mgmt_net_cfg(const char *name)
 	Mgmt__GetAttachInfoResp *resp;
 
 	/* Query the agent for the CaRT network configuration parameters */
-	rc = get_attach_info(name, false /* all_ranks */, &info, &resp);
+	rc = get_attach_info(name, true/* all_ranks */, &info, &resp);
```

Comment:
if we have 100 clients processes on 1 node, will this generate 100 drpc calls from the agent on that node to the server?

Reply:
Per my understanding it should not as agent is supposed to only let 1 through and cache the rest. @kjacque can you verify this for us?

Reply:
The agent caches the first response, yes.